image_proc/src/resize.cpp

```diff
@@ -63,7 +63,7 @@ ResizeNode::ResizeNode(const rclcpp::NodeOptions & options)
         }
       }
 
-      image_topic_ = camera_namespace_ + "/image";
+      image_topic_ = camera_namespace_ + "/camera";
```

Comment:
Why change the default to `/camera`? Isn't `image` more standard?

Reply:
This was a mistake on my end, I'll be fixing this